prompt-service/src/unstract/prompt_service/prompt_service_file_helper.py

```diff
@@ -0,0 +1,31 @@
+from typing import Union
+
+from unstract.prompt_service.constants import FileStorageKeys, FileStorageType
+from unstract.prompt_service.env_manager import EnvLoader
+from unstract.sdk.file_storage import (
+    FileStorageProvider,
+    PermanentFileStorage,
+    SharedTemporaryFileStorage,
+)
+
+
+class PromptServiceFileHelper:
```

Comment:
Since we are anyway writing this new class, we could try using the env_helper in sdk for this?

Reply:
Can we directly use EnvHelper.get_storage() here?

Reply:
@gaya3-zipstack These are files to be deleted after using `env_helper` Removing these

Reply:
@gaya3-zipstack These are files to be deleted after using `env_helper` Removing these